README.md

```diff
@@ -69,30 +70,22 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
 
 4. Start running your own analysis!
 
-    * Typical command for RNA-seq analysis:
-
-        ```bash
-        nextflow run nf-core/rnaseq \
-            --input samplesheet.csv \
-            --genome GRCh37 \
-            -profile <docker/singularity/podman/conda/institute>
-        ```
+    ```console
+    nextflow run nf-core/rnaseq \
+        --input samplesheet.csv \
+        --genome GRCh37 \
+        -profile <docker/singularity/podman/conda/institute>
+    ```
 
-    * Typical command for downloading public data:
+    * An executable Python script called [`fastq_dir_to_samplesheet.py`](https://github.com/nf-core/rnaseq/blob/master/bin/fastq_dir_to_samplesheet.py) has been provided if you would like to auto-create an input samplesheet based on a directory containing FastQ files **before** you run the pipeline (requires Python 3 installed locally) e.g.
 
-        ```bash
-        nextflow run nf-core/rnaseq \
-            --public_data_ids ids.txt \
-            -profile <docker/singularity/podman/conda/institute>
+        ```console
+        ~/.nextflow/assets/nf-core/rnaseq/bin/fastq_dir_to_samplesheet.py <FASTQ_DIR> samplesheet.csv
```

Comment:
That would require a `nextflow pull` first. I would either add that, or use something like  `curl -s https://raw.githubusercontent.com/nf-core/rnaseq/dev/bin/fastq_dir_to_samplesheet.py | python <FASTQ_DIR> samplesheet.csv`

Reply:
Fixed https://github.com/nf-core/rnaseq/pull/660/commits/76dfe0c144fa6fe450739dad940b2f3f29176536